tests/travis/install-mssql-cs.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
Instead of introducing a new script by duplicating the code, it would be better to parameterize the existing one. See:
https://github.com/doctrine/dbal/blob/9093b7b8cd5bbd43695e75c1a5500c53e63b13a5/.travis.yml#L190
https://github.com/doctrine/dbal/blob/9093b7b8cd5bbd43695e75c1a5500c53e63b13a5/tests/travis/docker-run-mysql-or-mariadb.sh#L44-L50